mappings/net/minecraft/block/entity/BeehiveBlockEntity.mapping

```diff
@@ -11,6 +11,7 @@ CLASS net/minecraft/class_4482 net/minecraft/block/entity/BeehiveBlockEntity
 	FIELD field_31313 MIN_OCCUPATION_TICKS_WITHOUT_NECTAR I
 	FIELD field_31314 ANGERED_CANNOT_ENTER_HIVE_TICKS I
 	FIELD field_31315 MIN_OCCUPATION_TICKS_WITH_NECTAR I
+	FIELD field_33570 KEY_TO_REMOVE Ljava/util/List;
```

Comment:
```suggestion
	FIELD field_33570 KEYS_TO_REMOVE Ljava/util/List;
```

Reply:
It would be better if we can describe why they should be removed though.

Reply:
Are these item-form incompatible block entity tag keys? Then `ITEM_INCOMPATIBLE_KEYS` or some better name (yeah, I think my suggested name is not fantastic) following that idea would be more descriptive.

Reply:
Wait, I forgot whose keys they are... Thought it was for BeeEntity, lemme see

Reply:
Yep I was right, it's for BeeEntity, and removes NBTs on `releaseBee`. Think they are ones that should not be kept when bees go inside hive

Reply:
```suggestion
	FIELD field_33570 BEE_KEYS_TO_REMOVE Ljava/util/List;
```
Guess this?